test/jdk/java/net/httpclient/http2/BadHeadersTest.java

```diff
@@ -23,10 +23,13 @@
 
 /*
  * @test
- * @bug 8303965
+ * @bug 8303965 8354276
  * @library /test/lib /test/jdk/java/net/httpclient/lib
  * @build jdk.httpclient.test.lib.http2.Http2TestServer jdk.test.lib.net.SimpleSSLContext
  * @run testng/othervm -Djdk.internal.httpclient.debug=true BadHeadersTest
+ * @summary This test verifies the behaviour of the HttpClient when presented
```

Comment:
Nit - the jtreg tag ordering guideline recommends adding `@summary` after the `@bug` line https://openjdk.org/jtreg/tag-spec.html#ORDER.